freqtrade/configuration/timerange.py

```diff
@@ -27,6 +30,34 @@ def __eq__(self, other):
         return (self.starttype == other.starttype and self.stoptype == other.stoptype
                 and self.startts == other.startts and self.stopts == other.stopts)
 
+    def subtract_start(self, seconds) -> None:
+        """
+        Subtracts <seconds> from startts if startts is set.
+        :param seconds: Seconds to subtract from starttime
+        :return: None (Modifies the object in place)
+        """
+        if self.startts:
+            self.startts = self.startts - seconds
+
+    def adjust_start_if_necessary(self, ticker_interval_secs: int, startup_candles: int,
+                                  min_date: arrow.Arrow) -> None:
+        """
+        Adjust startts by <startup_candles> candles.
+        Applies only if no startup-candles have been available.
+        :param ticker_interval_secs: Ticker interval in seconds e.g. `timeframe_to_seconds('5m')`
+        :param startup_candles: Number of candles to move start-date forward
+        :param min_date: Minimum data date loaded. Key kriterium to decide if start-time
+                         has to be moved
+        :return: None (Modifies the object in place)
+        """
+        if (not self.starttype or (startup_candles
+                                   and min_date.timestamp >= self.startts)):
+            # If no startts was defined, or test-data starts at the defined test-date
+            logger.warning("Moving start-date by %s candles to account for startup time.",
```

Comment:
no, debug definitely not - we could convert it to info... 

However to me, it's a warning - this covers the case where not enough "startup canles" are available - and i think users should know that their requested backtest range was not fullfilled.
It falls into the same range than "Missing data at start..." and "Missing data at end..." - both are warnings, too.

If i have data since january, but backtest starting with march - i will never see this message.